Web.md

```diff
@@ -202,6 +202,92 @@ Given these values, to process a potential WebAssembly response:
       _reason_.
 1. Return _returnValue_.
 
+### Structured serialization of a `WebAssembly.Memory`
+
+A `WebAssembly.Memory` is a
+[serializable object](https://html.spec.whatwg.org/multipage/infrastructure.html#serializable-objects), and will require changes to the
+[HTML Standard](https://html.spec.whatwg.org/multipage/infrastructure.html#serialization-steps) as follows:
+
+[StructuredSerializeInternal](https://html.spec.whatwg.org/multipage/infrastructure.html#structuredserializeinternal) adds a new step after step 12:
+
+* Otherwise, if *value* has a [[WebAssembly.Memory]] internal slot, then:
+  1. Let *size* be *value*.[[BufferObject]].[[ArrayBufferByteLength]].
+  2. Let *maximum* be the value of *value*.[[WebAssembly.Memory]]'s `max` field (which is an optional u32).
+  3. If *value*.[[BufferObject]] is a Shared Data Block, then:
+     1. If *forStorage* is true, then throw a
+        "[`DataCloneError`](https://heycam.github.io/webidl/#datacloneerror)"
+        [`DOMException`](https://heycam.github.io/webidl/#dfn-DOMException).
+     2. Otherwise,
+        set *serialized* to { [[Type]]: "WebAssembly.Memory",
+        [[ArrayBufferData]]: *value*.[[BufferObject]].[[ArrayBufferData]],
+        [[ArrayBufferByteLength]]: *size*,
+        [[MemoryMaximum]]: *maximum*,
+        [[AgentCluster]]: the [current Realm Record](https://tc39.github.io/ecma262/#current-realm)'s corresponding [agent cluster](https://tc39.github.io/ecma262/#sec-agent-clusters) }.
+  4. Otherwise throw a
+     "[`DataCloneError`](https://heycam.github.io/webidl/#datacloneerror)"
+     [`DOMException`](https://heycam.github.io/webidl/#dfn-DOMException).
+
+[StructuredDeserialize](https://html.spec.whatwg.org/multipage/infrastructure.html#structureddeserialize) adds a new step after step 12:
+
+* Otherwise, if *serialized*.[[Type]] is "WebAssembly.Memory", then:
+  1. Assert [IsSharedArrayBuffer](https://tc39.github.io/ecma262/#sec-issharedarraybuffer)(*serialized*.[[ArrayBufferData]]) is true.
```

Comment:
You can't use IsSharedArrayBuffer on a data block, only on JavaScript objects. Instead, this should be "Assert: *serialized*.[[ArrayBufferData]] is a Shared Data Block.